src/level.py

```diff
@@ -24,8 +24,9 @@
 
 class Level:
 
-    def __init__(self, tmx_maps, character_frames, level_frames, overlay_frames, font, sounds, switch):
+    def __init__(self, game, tmx_maps: MapDict, character_frames, level_frames, overlay_frames, font, sounds, switch):
```

Comment:
It seems like you added the 'game' parameter but didn't pass it in main.py

Reply:
I definitely did, it must have disappeared when I resolved the merge conflicts. Going to change that